chemprop/cli/train.py

```diff
@@ -890,11 +889,10 @@ def train_model(
             else:
                 predss = trainer.predict(dataloaders=test_loader)
 
-            preds = torch.concat(predss, 0).numpy()
-            if isinstance(model.predictor, MveFFN):
-                preds = np.split(preds, 2, axis=1)[0]
-            elif isinstance(model.predictor, EvidentialFFN):
-                preds = np.split(preds, 4, axis=1)[0]
+            preds = torch.concat(predss, 0)
+            if model.predictor.n_targets > 1:
+                preds = preds[..., 0]
+            preds = preds.numpy()
```

Comment:
just confirming my understanding here: this is because our outputs have shape $(n, t)$ in the case of no uncertainty and $(n, t, u)$ in the case of multiple uncertainty targets, and we only want to return the 0th uncertainty target?

Reply:
Yes